pkg/app/web/src/components/application-detail-page/application-state-view/index.tsx

```diff
@@ -22,6 +22,16 @@ import {
   selectHasError,
 } from "~/modules/applications-live-state";
 import { KubernetesStateView } from "./kubernetes-state-view";
+import { CloudRunStateView } from "./cloudrun-state-view";
+
+const isDisplayLiveState = (
+  app: Application.AsObject | undefined
+): app is Application.AsObject => {
```

Comment:
nits, it looks not so true 👀  Just want to confirm, do you mean that only count an app as of kind `Application.AsObject` in case of `app?.kind` is k8s or cloud run. If it's on purpose, it's good to me 👍  If not, should use `boolean` as the return value type.

Reply:
Oh, I see. Let me fix it!